crates/sequencer/src/state.rs

```diff
@@ -20,18 +20,18 @@ use crate::serde::SerializableState;
 pub type ContractStorageKey = (ContractAddress, StorageKey);
 
 /// Generic state structure for the sequencer.
-/// The use of `FxHashMap` allows for a better performance.
+/// The use of `HashMap` implementation from hashbrown allows for a better performance.
 /// This hash map is used by rustc. It uses a non cryptographic hash function
 /// which is faster than the default hash function. Think about changing
 /// if the test sequencer is used for tests outside of ef-tests.
 /// See [rustc-hash](https://crates.io/crates/rustc-hash) for more information.
```

Comment:
Can you update the comment, removing anything about rustc-hash with comments about the hashbrown hashmap? You should be able to find information online about why it is more performant than the `FxHashMap` and also better than the default `std` hashmap.